src/templates/deployment.yaml

```diff
@@ -45,3 +45,13 @@ spec:
           env:
             - name: EULA
               value: {{ .Values.eula | squote }}
+          livenessProbe:
+            exec:
+              command: ["mc-monitor", "status"]
+            periodSeconds: 5
+            timeoutSeconds: 3
+          startupProbe:
+            exec:
+              command: ["mc-monitor", "status"]
+            initialDelaySeconds: 10
```

Comment:
I think this isn't relevant for startup probes.
The only consequence such setting may have is a negative one: i.e., preventing detecting that the server is up in less than 10 seconds (which is not impossible if CPU is insane or fine-tuning allow that).
Stated differently: I fear we have a take an opinion for no value.

Reply:
Such setting may be relevant for `livenessProbe` and `readinessProbe` in contexts where no `startupProbe` are defined.
That's my guess at least.